cocos/2d/CCLabel.h

```diff
@@ -749,9 +762,8 @@ class CC_DLL Label : public Node, public LabelProtocol, public BlendProtocol
     Color4F _textColorF;
 
     QuadCommand _quadCommand;
-    CustomCommand _customCommand;
-    CustomCommand _customCommandOutLine;
-    CustomCommand _customCommandShadow;
+
+    std::vector<BatchCommand> _batchCommands;
```

Comment:
_batchCommands does not seem to be set?

Reply:
It is set in `Label::draw` though `updateShaderProgram`
```cpp
            if (_batchCommands.size() != _batchNodes.size())
            {
                _batchCommands.resize(_batchNodes.size());
                updateShaderProgram();
            }
```